Themes/default/languages/Help.english.php

```diff
@@ -403,6 +403,10 @@
 	</ul>';
 
 $helptxt['autoFixDatabase'] = 'This will automatically fix broken tables and resume as if nothing happened. This can be useful, because the only way to fix it is to REPAIR the table, and this way your forum won\'t be down until you notice. It does email you when this happens.';
+$helptxt['pgFulltextSearch'] = 'This setting allow your to define the language wich is used for the fulltext search of postgresql, the language settings are independent from your smf settings and not related to the installed language for smf. <br/>
+When you change the language you need to rebuild the fulltext search index to find under Search -> Search Method. (Fulltext Index)<br/>
+The listed language based on the postgresql installation,
+when you miss a language ask your database admin.';
```

Comment:
How about the following?
```
$helptxt['pgFulltextSearch'] = 'This setting defines the language to use for PostgreSQL's fulltext search. Choose the language that most closely matches the language your forum actually uses. If your forum's language is not listed, or if your forum is multi-lingual, choose the "simple" option. <br/>
This setting is independent of your main SMF language settings and not related to the installed language for SMF. <br/>
When you change this setting you need to [url=' . $scripturl . '?action=admin;area=managesearch;sa=method]rebuild the fulltext search index[/url].<br/>
If a language you need is not listed, ask your database admin to install PostgreSQL language support for that language.';
```

You'll need to add $scripturl to the globals at the start of this file, of course.